### PR TITLE
Add Developer Section / How to Commit from cwiki.apache.org

### PR DESCRIPTION
**Pull request by bessbd**

### CODE REVIEW

File: dev-docs/HowToCommit.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+-   **Commit the patch:** If the patch meets review expectations and is
+    well tested, it can be committed to the source control. Make sure
+    that the patch applies cleanly to the latest revision and if not,
+    request the patch be rebased accordingly. Once ready for commit, the
+    commit message should have the following format:
+
+    ```
+    Flume-XXX. Brief description of the problem.
+
```
Comment (bessbd) [inline dev-docs/HowToCommit.md:52]: Maybe we should add the "Reviewers:" line here (and "This closes #x") if it's a pull request.

**Review by simonati — COMMENTED** [review]
Missing section :"This guide highlights the process and responsibilities of committers on Flume project. Note: Just like contributors, committers must follow the same guidelines for contributions as highlighted in the How to Contribute guide." 
I checked content, links, formatting against existing cwiki. TOC is missing but seems like it is down to github https://github.com/isaacs/github/issues/215
Otherwise looks good to me.

**Comment by bessbd** [comment]
Thank you, @satiatcloudera 

Good catch!
I'll add those lines and commit it.

**Review by simonati — APPROVED** [review]
LGTM

**Comment by bessbd** [comment]
I'm about to commit this.

**Pull request closed without merging**